### DISCUSSION TITLE
alternative locations for models

### DISCUSSION THREAD

**Discussion by rpillala** [vladmandic/sdnext#327] [Q&A] [answered]
I would like to specify locations for checkpoints, loras, hypernetworks, and embeddings. A111, this is accomplished with --ckpt-dir (etc) pointing to the locations, but I don't see that among the available flags in this repo. Is there a way to do it? I see I can specify a location for a specific with --ckpt but not a location for a set of checkpoints. The checkpoints are large compared to the rest of the installation, and I keep them on a spinning disk and not the SSD. 

Thanks!

**Comment by vladmandic** [comment, marked as answer]
It's in UI settings.

**Comment by rpillala** [comment]
I failed to consider this because without a model in the default location, I never got to the UI settings. But on the other hand maybe it's in ui-config.json let me look.

**Reply by vladmandic** [reply]
You can use  - - ckpt to force load a model
